action.yml

```diff
@@ -16,8 +16,12 @@ inputs:
     required: true
   username:
     required: true
-  password:
-    required: true
```

Comment:
Is this considered a breaking change? Like If I pass an argument that's not listed in the `inputs` field, will the action throw an error?

Reply:
I don't think so, but i am not sure.
I tried to pass extra argument to the action and it passed with success, but left some warnings below about extra argument

Reply:
![Screenshot from 2023-06-28 15-13-40](https://github.com/neondatabase/create-branch-action/assets/3032829/0dc021f9-ccd2-4afe-8cab-0b52065a8600)